app/messages/Messages.scala

```diff
@@ -963,9 +963,14 @@ object Messages {
       welsh   = s"""Defnyddiwch wasanaeth Text Relay UK os na allwch glywed na siarad dros y ffôn. Deialwch <strong>18001</strong> ac yna <strong>0345 300 3900</strong>. Dysgwch ragor am hyn ar <a href="$linkUrl" rel="noreferrer noopener" target="_blank">wefan Text Relay UK (yn agor tab newydd)</a>."""
     )
 
-    def `If a health condition or personal circumstances make it difficult to contact us...`(linkUrl: String): Message = Message(
-      english = s"""<strong>If a health condition or personal circumstances make it difficult to contact us, read our guidance <a class="govuk-link" href="$linkUrl" rel="noreferrer noopener" target="_blank">Get help from HMRC if you need extra support (opens in new tab)</a>.</strong>""",
-      welsh   = s"""<strong>Os yw cyflwr iechyd neu amgylchiadau personol yn ei gwneud hi’n anodd i chi gysylltu â ni, darllenwch ein harweiniad ynghylch <a href="$linkUrl" rel="noreferrer noopener" target="_blank">‘Cael help gan CThEF os oes angen cymorth ychwanegol arnoch’ (yn agor tab newydd)</a>.</strong>"""
+    def `If a health condition or personal circumstances make it difficult to contact us...`: Message = Message(
+      english = s"""If a health condition or personal circumstances make it difficult to contact us""",
```

Comment:
just for future reference, you could have written
```scala
english = "If a health..."
```
since nothing is being interpolated and you're including any special characters in there that would otherwise need escaping